2-mandatory/6-journey-planner.js

```diff
@@ -122,7 +128,9 @@ function getLocationName() {}
   Advanced challange: try to use arrow function when invoking an array method.
 */
 function journeyPlanner(locations, transportMode) {
-  // Implement the function body
+  return locations
+    .filter((location) => location.includes(transportMode))
+    .map((location) => location[0]);
 }
```

Comment:
This is fine Andrius, you have already defined a funcition above called getLocationName, can you think how you can use this function in line 133?